scene/3d/skeleton_modifier_3d.cpp

```diff
@@ -129,10 +129,13 @@ void SkeletonModifier3D::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("set_influence", "influence"), &SkeletonModifier3D::set_influence);
 	ClassDB::bind_method(D_METHOD("get_influence"), &SkeletonModifier3D::get_influence);
 
+	ClassDB::bind_method(D_METHOD("process_modification"), &SkeletonModifier3D::process_modification);
+
```

Comment:
```suggestion
```

After giving it some thought, I conclude that it should not be added at this stage. An example would probably require the following code (which is still not enough):

SkeletonModifier3D
```cpp
void SkeletonModifier3D::process_modification() {
	ERR_FAIL(!get_skeleton()->is_processing_modifiers);
	if (!active) {
		return;
	}
	_process_modification();
	emit_signal(SNAME("modification_processed"));
}
```

Skeleton3D
```cpp
bool processing_modifiers = false;

bool Skeleton3D::is_processing_modifiers() {
	return processing_modifiers ;
}

void Skeleton3D::_process_modifiers() {
	for (const ObjectID &oid : modifiers) {
		processing_modifiers = true;
		emit_signal(SNAME("modification_process_started"));

		Object *t_obj = ObjectDB::get_instance(oid);
		if (!t_obj) {
			continue;
		}
		SkeletonModifier3D *mod = cast_to<SkeletonModifier3D>(t_obj);
		if (!mod) {
			continue;
		}
		real_t influence = mod->get_influence();
		if (influence < 1.0) {
			LocalVector<Transform3D> old_poses;
			for (int i = 0; i < get_bone_count(); i++) {
				old_poses.push_back(get_bone_pose(i));
			}
			mod->process_modification();
			LocalVector<Transform3D> new_poses;
			for (int i = 0; i < get_bone_count(); i++) {
				new_poses.push_back(get_bone_pose(i));
			}
			for (int i = 0; i < get_bone_count(); i++) {
				if (old_poses[i] == new_poses[i]) {
					continue; // Avoid unneeded calculation.
				}
				set_bone_pose(i, old_poses[i].interpolate_with(new_poses[i], influence));
			}
		} else {
			mod->process_modification();
		}
		force_update_all_dirty_bones();
	}
}
```
However, as you can see, I assume there is no way to safely interrupt a custom process here at this stage.